### DISCUSSION TITLE
How do I switch mode?

### DISCUSSION THREAD

**Discussion by Vadimluck** [innightwolfsleep/llm_telegram_bot#70] [Q&A]
How do I switch from chat to notepad?

**Comment by innightwolfsleep** [comment]
open telegram_config.cfg
and set "bot_mode=notebook"
p.s. notebook mode is poor as for me. This is mode without character names, you controled context you own. Also, usu some "empty" character to work in notebook mode.
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/20921671/7f8dbe1f-2cdd-4158-ba83-d902af4692ce)

**Comment by Vadimluck** [comment]
Thanks, I probably won't use it, but I have some models that work better in notebook mode.

**Reply by Vadimluck** [reply]
Where do you want to add an empty symbol?

**Reply by innightwolfsleep** [reply]
I mean - context suitable for notebook mode.

**Reply by Vadimluck** [reply]
Does this need to be added to some txt file or web interface?

**Reply by innightwolfsleep** [reply]
No, just run it like ordinary mode

**Reply by Vadimluck** [reply]
If I set bot_mode=notebook and just run it, it somehow does not write answers, only my questions.
